packages/integrations/cloudflare/README.md

```diff
@@ -191,6 +191,42 @@ export default defineConfig({
 });
 ```
 
+## WASM module imports
+
+`wasmModuleImports: boolean`
+
+Cloudflare has native support for importing `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration). To enable importing them as modules in the cloudflare build and astro dev server, add `wasmModuleImports: true` to your config.
+
+```diff
+import {defineConfig} from "astro/config";
```

Comment:
```suggestion
// astro.config.mjs
import {defineConfig} from "astro/config";
```
We do add a file name whenever it's easy to do so, so that it's clear to someone just skimming code samples where this code is!